cpp/dolfinx/fem/CoordinateElement.h

```diff
@@ -93,6 +93,19 @@ class CoordinateElement
   void tabulate(int nd, std::span<const T> X, std::array<std::size_t, 2> shape,
                 std::span<T> basis) const;
 
+  /// @brief Permute a list of DOF indices associated with the closure of a cell
+  /// sub-entity.
```

Comment:
Could you add some more explanation to the docstring on what this function does?

Helpful to include a mathematical description too, e.g. https://github.com/FEniCS/dolfinx/blob/ab79530e1ae60a646002609b510e8392cc09e213/cpp/dolfinx/fem/FiniteElement.h#L663.